Source/WebCore/css/ImmutableStyleProperties.cpp

```diff
@@ -41,8 +41,8 @@ ImmutableStyleProperties::ImmutableStyleProperties(const CSSProperty* properties
     auto* valueArray = bitwise_cast<PackedPtr<CSSValue>*>(this->valueArray());
     for (unsigned i = 0; i < length; ++i) {
         metadataArray[i] = properties[i].metadata();
-        auto* value = properties[i].value();
-        valueArray[i] = value;
+        RefPtr value = properties[i].value();
+        valueArray[i] = value.get();
```

Comment:
Rather than calling `get()` then manually calling `ref()`, we should call `leakRef()` instead.